src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.cs

```diff
@@ -238,29 +238,31 @@ private IMember GetValueFromFString(FString fString)
                 return null;
             }
 
-            IPythonInstance instance = null;
             var m = GetValueFromExpression(expr.Target);
-            if (m is IPythonType typeInfo) {
-                var member = typeInfo.GetMember(expr.Name);
-                // If container is class/type info rather than the instance, then the method is an unbound function.
-                // Example: C.f where f is a method of C. Compare to C().f where f is bound to the instance of C.
-                if (member is PythonFunctionType f && !f.IsStatic && !f.IsClassMethod) {
-                    f.AddReference(GetLocationOfName(expr));
-                    return f.ToUnbound();
-                }
-                instance = new PythonInstance(typeInfo);
+            if(m == null) {
+                return UnknownType;
             }
 
-            instance = instance ?? m as IPythonInstance;
-            var type = m?.GetPythonType(); // Try inner type
-
+            var type = m.GetPythonType();
             var value = type?.GetMember(expr.Name);
             type?.AddMemberReference(expr.Name, this, GetLocationOfName(expr));
 
             if (type is IPythonModule) {
                 return value;
             }
 
+            IPythonInstance instance = null;
```

Comment:
Change to fix the issue when standalone function could become an unbound method with declaring type of null. Does not really affect completions, but present problems when restoring function from the database.